lib/engine/game/g_18_we/map.rb

```diff
@@ -0,0 +1,301 @@
+            ['B7'] => 'path=a:1,b:5',
+            ['C8'] => 'path=a:2,b:4',
+            ['D7'] => 'path=a:1,b:4',
+            # London Channel crossings
+            ['F7'] => 'path=a:0,b:2,track:dual;path=a:2,b:4,track:dual;path=a:2,b:5,track:dual',
+            ['E8'] => 'path=a:0,b:3,track:dual;path=a:3,b:5,track:dual',
+          },
+          white: {
+            # Germanic Region:
+            ['L7'] => 'frame=color:#59b577;city=revenue:0;label=Z',
+            %w[
+                    K2 J5 N5 H7 L9 M10 H11 J13
+                ] => 'frame=color:#59b577;city=revenue:0;label=Y',
+            %w[
+                    O4 L5 G8 H9 I10 H13
+                ] => 'frame=color:#59b577;city=revenue:0',
+            %w[
+                    J3 I6 K6 M6 O6 J7 N7 I8 K8 M8 J9 K10 J11 I12
+                ] => 'frame=color:#59b577;town=revenue:0',
+
+            # Southeastern Region:
+            ['I16'] => 'frame=color:#fbeb66;city=revenue:0;label=Z',
+            %w[
+                    O8 L11 M14 O14 K16 I18 J21
+                ] => 'frame=color:#fbeb66;city=revenue:0;label=Y',
+            %w[
+                    N9 O10 K12 M12 L15 J17
+                ] => 'frame=color:#fbeb66;city=revenue:0',
+            %w[
+                    N11 O12 L13 N13 I14 K14 H15 J15 N15 M16 H17 K18 H19
+                    J19 K20
+                ] => 'frame=color:#fbeb66;town=revenue:0',
+
+            # French Region:
+            ['F11'] => 'frame=color:#b14bd3;city=revenue:0;label=Z',
+            %w[
+                    C12 E12 C16 G16 G20
+                ] => 'frame=color:#b14bd3;city=revenue:0;label=Y',
+            %w[
+                    E10 B11 G14 F15 E16 D17 G18 D19
+                ] => 'frame=color:#b14bd3;city=revenue:0',
+            %w[
+                    F9 G10 D11 G12 D13 F13 C14 E14 D15 F17 C18 E18 F19
+                    E20
+                ] => 'frame=color:#b14bd3;town=revenue:0',
+          },
+        }.freeze
+
+        LOCATION_NAMES = {
+          'K0' => 'Scandinavia',
+          'J1' => 'Skaggerak',
+          'K2' => 'København',
+          'L3' => 'Denmark Sound',
+          'N3' => 'Baltic Sea',
+          'P5' => 'St. Petersburg',
+          'O4' => 'Konigsburg',
+          'H5' => 'North Sea',
+          'J5' => 'Hamburg',
+          'L5' => 'Stettin',
+          'N5' => 'Danzig',
+          'E6' => 'London',
+          'H7' => 'Amsterdam',
+          'L7' => 'Berlin',
+          'A8' => 'Greater UK',
+          'G8' => 'Brussels',
+          'O8' => 'Warsaw',
+          'H9' => 'Essen',
+          'L9' => 'Dresden',
+          'N9' => 'Łódź',
+          'P9' => 'Poland & Ukraine',
+          'C10' => 'English Channel',
+          'E10' => 'Rouen',
+          'I10' => 'Frankfurt',
+          'M10' => 'Breslau',
+          'O10' => 'Krakow',
+          'B11' => 'Brest',
+          'F11' => 'Paris',
+          'H11' => 'Köln',
+          'L11' => 'Praha',
+          'C12' => 'Nantes',
+          'E12' => 'Orleans',
```

Comment:
Orléans. It doesn't have the acute accent on the 18WE images on BGG, but generally local spellings are being used, so this would be more consistent.

Reply:
As you say, this commit currently reflects the existing board for the game. I have asked Karl as designer for his input on changing things. As well as the two you have highlighted, there are 11 other locations which could be more "localised" - the most striking being Constantinople (which could be rendered as Konstantiniyye or Istanbul).

Reply:
This is something that's always easy to change later, so I don't think we need to hold up pre-alpha progress for it.